packages/frontend/src/components/quiz/QuizLocation/QuizLocation.tsx

```diff
@@ -0,0 +1,20 @@
+import { list as listStyle } from "./QuizLocation.css";
+import QuizLocationItem from "./QuizLocationItem";
+
+interface QuizLocationProps {
+  items: string[];
+}
+
+export default function QuizLocation({ items }: QuizLocationProps) {
+  const { length } = items;
+
+  const children = items.map((item, index) => (
+    <QuizLocationItem key={item} title={item} last={isLast(index, length)} />
+  ));
+
+  return <ol className={listStyle}>{children}</ol>;
+}
+
+function isLast(index: number, length: number) {
+  return index === length - 1;
+}
```

Comment:
아이콘 검사가 맘에 걸리시면 이건 어때요??

```suggestion
export default function QuizLocation({ items }: QuizLocationProps) {
  const { length } = items;

  return (
    <ol className={listStyle}>
      {items.map((item, index) => (
        <li key={item} className={flexAlignCenter}>
          <span>{item}</span>
          {!isLast(index, length) && <ChevronRight />}
        </li>
      ))}
    </ol>
  );
}

function ChevronRight() {
  return (
    <span className={iconStyle}>
      <BsChevronRight size={10} />
    </span>
  );
}
```

Reply:
오 좋은데요? 네 이렇게 변경할게요!